### PR TITLE
Fix flakiness in test_futimens_rawfs

### PR DESCRIPTION
**Pull request by sbc100**
This newly added test was flaky because noderawfs wasn't handling the value of -1 correctly here.

### CODE REVIEW

File: src/library_noderawfs.js [hunk 1]
```diff
@@ -99,7 +99,17 @@ addToLibrary({
       var stream = FS.getStreamChecked(fd);
       fs.ftruncateSync(stream.nfd, len);
     },
-    utime(path, atime, mtime) { fs.utimesSync(path, atime/1000, mtime/1000); },
+    utime(path, atime, mtime) {
+      // -1 here for atime or mtime means UTIME_OMIT was passed.  Since node
+      // doean't support this concept we need first find the existing timestamps
```
Comment (kripken) [inline src/library_noderawfs.js:104]:
```suggestion
      // doesn't support this concept we need to first find the existing timestamps
```

**Review by kripken — APPROVED** [review]

**Comment by brendandahl** [comment]
I've noticed this test failing quite a bit lately. Can we merge this in?

**Comment by sbc100** [comment]
IIRC there was some reason I didn't land this already.. perhaps that it didn't actually fix the flakeyness.. I'm not sure.

**Review by Fart10 — COMMENTED** [review]
😜

**Comment by sbc100** [comment]
> 😜

Thanks for the feedback!

**Pull request merged**